scripts/talos/README.md

```diff
@@ -0,0 +1,55 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot which runs automatically on the latest branch of Psammead. Talos was created to make life easier for developers, with Talos they do not need to keep track of their dependencies, it saves time, makes sure dependencies are up to date and prevents conflicts. Furthermore, it bumps everything packages in 2 PRs, therefore it is quicker to review PRs, whereas Dependabot would do the same in more than 2 PRs.
```

Comment:
```suggestion
Talos is a bumper bot which runs automatically on the latest branch of Psammead. Talos was created to make life easier for developers, with Talos they do not need to keep track of their dependencies, it saves time, makes sure dependencies are up to date and it will prevent a duplicate package bundle from existing in external applications when multiple Psammead packages are used. Furthermore, it bumps everything packages in 2 PRs, therefore it is quicker to review PRs, whereas Dependabot would do the same in more than 2 PRs.
```

Reply:
`prevent a duplicate package bundle from existing in external applications when multiple Psammead packages are used.` i added this

Reply:
https://github.com/bbc/simorgh/blob/latest/webpack.config.client.js#L91-L95
we use inspectpack in Simorgh to check for duplication, you could include a description of this